config/default.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from fsd_tech import configclass
+
+
+@configclass
+class DefaultConfig:
+
+    FLASK_ENV = os.environ.get("FLASK_ENV", "flask_env")
+    GOV_NOTIFY_API_KEY = os.environ.get(
```

Comment:
I wonder if we should throw a key error if this is not set rather than defaulting to an invalid value, as this key is an essential requirement for this app, maybe a Key Error exception makes more sense?

Reply:
I'd agree, general principle of default config should be to throw key errors when required variables aren't set.